packages/components/src/buttons/src/Button.module.css

```diff
@@ -0,0 +1,277 @@
+.hop-button {
+    --hop-button-component-padding: 0 var(--hop-space-inset-md);
+    --hop-button-component-border-radius: var(--hop-shape-rounded-md);
+    --hop-button-component-focus-ring-color: var(--hop-primary-surface-hover); /* TODO: place to centralize tokens like this? */
+    --hop-button-component-md-height: 2.5rem;
+    --hop-button-component-sm-height: 2rem;
+
+    /* Primary */
+    --hop-button-component-primary-color: var(--hop-primary-text-strong);
+    --hop-button-component-primary-color-hover: var(--hop-primary-text-strong-hover);
+    --hop-button-component-primary-color-pressed: var(--hop-primary-text-active);
+    --hop-button-component-primary-color-disabled: var(--hop-neutral-text-disabled);
+    --hop-button-component-primary-background-color: var(--hop-primary-surface-strong);
+    --hop-button-component-primary-background-color-hover: var(--hop-primary-surface-hover);
+    --hop-button-component-primary-background-color-pressed: var(--hop-primary-surface-active);
+    --hop-button-component-primary-background-color-disabled: var(--hop-neutral-surface-disabled);
+    --hop-button-component-primary-border: 0;
```

Comment:
Do we need to specify 0 for all these?

Reply:
you mean just have --hop-button-component-primary-border?

Reply:
i tried to have similar tokens for all variant. The primary variant does not have any borders, so i figured i should keep the tokens, but set them to 0

Reply:
You could use the variable fallback.

Reply:
I changed my mind, it's better the way you did it, for consistency and future changes.